ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4409,7 +4417,130 @@ HWDDC:
   implements:
   - CONTROL
 
+# New abstracts for MFA BV100 HVAC
+CFDPM:
+  description: "Carbon filter pressure monitoring."
+  is_abstract: true
+  uses:
+  - carbon_filter_differential_pressure_sensor
+  implements:
+  - MONITORING
+ 
+VOADM2X:
+  description: "Variable outside air damper monitoring."
```

Comment:
add to description: "[...] where there are two separate, equal sets of dampers that operate in conjunction."